Moonshot/player/player.gd

```diff
@@ -38,21 +38,28 @@ var safety = false
 var dead = false
 var priority_animations = ["stagger", "dodge"]
 
+var mdiff : Vector2
+var mouse : Vector2
+var msspd : float
```

Comment:
what is `msspd`?

Reply:
unused variable

Reply:
it was for a different mouse implementation, which i cut out for this PR, but didnt cut out all the bits